packages/myst-cli/src/process/file.ts

```diff
@@ -65,14 +65,15 @@ export async function loadNotebookFile(
 ): Promise<LoadFileResult> {
   const vfile = new VFile();
   vfile.path = file;
-  const mdast = await processNotebook(session, file, content);
-  const { frontmatter, identifiers } = getPageFrontmatter(
+  const { mdast, frontmatter: nbFrontmatter } = await processNotebookFull(session, file, content);
+  const { frontmatter: cellFrontmatter, identifiers } = getPageFrontmatter(
     session,
     mdast,
     vfile,
     opts?.preFrontmatter,
     opts?.keepTitleNode,
   );
+  const frontmatter = { ...nbFrontmatter, ...cellFrontmatter };
```

Comment:
@fwkoch is there a better way to merge this?

Reply:
Could use: 
```suggestion
  const frontmatter = fillProjectFrontmatter(cellFrontmatter, nbFrontmatter, { /** validation opts */ };
```

That should work fine with `PageFrontmatter` too. Main difference is that it combines some fields if they are defined in both objects, rather than just overriding.

Reply:
*That* is what I was hoping for. Thanks @fwkoch